docker/generic-worker-config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "accessToken": "j2Z6zW2QSLehailBXlosdw9e2Ti8R_Qh2M4buAEQfsMA",
+  "clientId": "static/generic-worker-compose-client",
+  "ed25519SigningKeyLocation": "/etc/generic-worker/ed25519_key",
+  "rootURL": "http://taskcluster",
+  "workerId": "generic-worker",
+  "provisionerId": "docker-compose",
+  "workerType": "generic-worker",
+  "livelogExecutable": "/usr/local/bin/livelog",
+  "taskclusterProxyExecutable": "/usr/local/bin/taskcluster-proxy",
+  "publicIP": "127.1.2.3"
```

Comment:
Does this work?

Reply:
most likely  :) I don't know how to test it. Builds were running, artifacts were uploaded.

Reply:
I'm trying to remember what it is used for. I think originally it was for serving livelogs using stateless dns server, which is no longer needed. I think maybe it is just part of the metadata that is signed by the chainOfTrust feature (the feature should really be called signedArtifacts).

Reply:
Looks like back in February when I started it, it was breaking on the artifact upload step ([jira comment](https://mozilla-hub.atlassian.net/browse/FCP-54?focusedCommentId=515032))